docs/connect.html

```diff
@@ -1042,21 +1003,20 @@ <h3><a id="connect_administration" href="#connect_administration">8.4 Administra
     You can use the REST API to view the current status of a connector and its tasks, including the ID of the worker to which each was assigned. For example, the <code>GET /connectors/file-source/status</code> request shows the status of a connector named <code>file-source</code>:
     </p>
 
-    <pre class="brush: json;">
-{
-    "name": "file-source",
-    "connector": {
-        "state": "RUNNING",
-        "worker_id": "192.168.1.208:8083"
-    },
-    "tasks": [
-        {
-        "id": 0,
-        "state": "RUNNING",
-        "worker_id": "192.168.1.209:8083"
-        }
-    ]
-}</pre>
+    <pre class="line-numbers"><code class="language-json">{
```

Comment:
The alignment seems broken for this code block:
![image](https://github.com/apache/kafka/assets/903615/164821a2-41bd-4cc9-bb2c-676838addecb)

Reply:
fixed now: 
![image](https://github.com/apache/kafka/assets/3337739/aadba3b2-c8eb-4967-89b0-d19255c18e8d)

(previous deleted message contained the wrong snapshot)

Reply:
Thanks!